internal/driver/html/graph.css

```diff
@@ -0,0 +1,7 @@
+#graph {
+    cursor: grab;
+}
+
+#graph:active {
+    cursor: grabbing;
+}
```

Comment:
Could you add missing newline at the end of this file?

Reply:
Done.